backend/src/main/resources/static/discovery-finder-openapi.yaml

```diff
@@ -1,3 +1,5 @@
+---
+# kics-scan ignore-block
```

Comment:
we wont ignore the openapi.yml

Reply:
I have fixed the KICS violations in the PR - https://github.com/eclipse-tractusx/sldt-digital-twin-registry/pull/145
So we can ignore this PR.